ansible/roles/operator-pipeline/templates/openshift/tasks/preflight.yml

```diff
@@ -19,20 +19,57 @@ spec:
       description: The Operator index image which includes the bundle
     - name: bundle_image
       description: The Operator bundle image under test
+    - name: service_account
+      description: The service account for the preflight config
+      default: "default"
+    - name: namespace
+      description: The namespace for the preflight config
+      default: "default"
```

Comment:
Please default this to `$(context.taskRun.namespace)` instead.

Reply:
Fixed